### DISCUSSION TITLE
ошибка 404

### DISCUSSION THREAD

**Discussion by shineanora** [community/community#123521] [Accessibility]
### Select Topic Area

Bug

### Body

Загрузила репозиторий как обычно, но когда хочу посмотреть свой сайт то выходит ошибка 404. Вроде все загрузила верно не понимаю в чем дело уже целый день потратила на это 
<img width="1470" alt="Снимок экрана 2024-05-12 в 23 42 04" src="https://github.com/community/community/assets/148863678/b20e596a-91fd-4f72-8cd1-8427e5c4468c">

**Comment by liker912** [comment]
@shineanora проверь все ли настройки есть в соответствии с документацией https://pages.github.com/#vanilla-step-1 для github pages.

**Comment by grandmas-favorite** [comment]
Hi @shineanora , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
